hr_timesheet_purchase_order/tests/test_create_not_product.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from freezegun import freeze_time
+
+from odoo.exceptions import UserError
+from odoo.tests.common import Form
+
+from .common_po_recurrence import TestTimesheetPOrecurrenceCommon
+
+
+class TestTimesheetPORecurrenceNotProduct(TestTimesheetPOrecurrenceCommon):
```

Comment:
Only one test on the all model? Would you please upped test coverage and add docstrings and assertions messages.

Reply:
I'll add docstrings and assertions messages.